aio/aio-proxy/aio_proxy/routes.py

```diff
@@ -65,9 +66,17 @@ async def near_point_endpoint(request):
     )
 
 
-@routes.get("/metadata/conventions_collectives")
+@routes.get("/idcc/metadata")
 async def conventions_collectives_endpoint(request):
     """
     Endpoint for serving the convention collective JSON file.
     """
     return get_metadata_cc_response()
+
+
+@routes.get("/idcc/{siren}")
+async def search_conventions_collectives_by_siren_endpoint(request):
+    """
+    Endpoint for searching conventions collectives by SIREN number.
+    """
+    return get_idcc_list_by_siren(request)
```

Comment:
There is an inconsistency between `get_metadata_cc_response`and `get_idcc_list_by_siren` names

Maybe use `get_ul_cc_response`

Reply:
also one could use `idcc` everywhere rather than either `cc` or `idcc`